### PR TITLE
159394833 Blocks layout

### PR DESCRIPTION
**Pull request by Kalela**
# What does this PR do?
- Sets up the layout UI for blocks.
- Sets up the functionality for the back button added.

## Followed designs
[https://projects.invisionapp.com/d/main/default/#/console/13559860/309161378/preview](https://projects.invisionapp.com/d/main/default/#/console/13559860/309161378/preview)

#### 10 inch tablet - Android API 27
screenshot_1533665209

#### 7 inch tablet - Android API 22
screenshot_1533809688

ezgif com-video-to-gif

### How should this be manually tested?
1. Clone the repo locally.
2. Set up an emulator or adb connection.
3. Run the app using android studio.

### What are the relevant pivotal tracker stories?
159394833
[An admin can view **blocks**](https://www.pivotaltracker.com/story/show/159394833)

### CODE REVIEW

**Comment by TheDancerCodes** [comment]
@Kalela 
Quick one, I have noticed that your screenshot displays `What building is this meeting room in?` while the designs read `What office is this meeting room in?`

Could you kindly look into this?

File: app/src/main/java/com/andela/mrm/room_setup/building/BuildingActivity.java [hunk 1]
```diff
@@ -30,6 +34,15 @@ protected void onCreate(Bundle savedInstanceState) {
                         .add(R.id.frame_layout_container, fragment)
                         .commit();
                    }
+
+        backButton = findViewById(R.id.blocksBackButton);
+        backButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = new Intent(BuildingActivity.this, CountryActivity.class);
```
Comment (idrisadetunmbi) [inline app/src/main/java/com/andela/mrm/room_setup/building/BuildingActivity.java]: @Kalela what happens when you press the hardware back button on the activity that this intent is sent to or started with?

**Comment by TheDancerCodes** [comment]
@Kalela 
Kindly sync with @Jak-Sidious and @meshnesh 

They are working to resolve the failing test issue.

**Comment by Kalela** [comment]
@idrisadetunmbi Kindly review the new changes.

**Pull request merged**